dashboard/test/controllers/ops/cohorts_controller_test.rb

```diff
@@ -13,20 +13,34 @@ class CohortsControllerTest < ::ActionController::TestCase
     end
 
     test 'district contact can add teachers to a cohort' do
-      sign_in @district.contact
+      @cohort.teachers << create(:teacher, district_id: @district.id)
+      @cohort.save!
+
+      new_cohorts_district = create(:cohorts_district, cohort: @cohort)
+      new_district = new_cohorts_district.district
+
+      @cohort.reload
+
+      # we already have a teacher from the old district
+      assert_equal 1, @cohort.teachers.count
+      assert_equal 2, @cohort.districts.count
+
+      sign_in new_district.contact
       #87054720 (part 1)
       #can click "Add Teacher" button to add a teacher
       assert_routing({ path: "#{API}/cohorts/1", method: :patch }, { controller: 'ops/cohorts', action: 'update', id: '1' })
 
-      teacher_params = @cohort.teachers.map {|teacher| {ops_first_name: teacher.name, email: teacher.email, id: teacher.id}}
-      teacher_params += [
-                         {ops_first_name: 'Laurel', ops_last_name: 'X', email: 'laurel_x@example.xx', district: @district.name, ops_school: 'Washington Elementary', ops_gender: 'Female'},
-                         {ops_first_name: 'Laurel', ops_last_name: 'Y', email: 'laurel_y@example.xx', district: @district.name, ops_school: 'Jefferson Middle School', ops_gender: 'Male'}
+      teacher_params = [
+                         {ops_first_name: 'Laurel', ops_last_name: 'X', email: 'laurel_x@example.xx', district: new_district.name, ops_school: 'Washington Elementary', ops_gender: 'Female'},
+                         {ops_first_name: 'Laurel', ops_last_name: 'Y', email: 'laurel_y@example.xx', district: new_district.name, ops_school: 'Jefferson Middle School', ops_gender: 'Male'}
                         ]
 
+      # we add these two new teachers and did not remove the old ones
       assert_difference('@cohort.reload.teachers.count', 2) do
         assert_difference('User.count', 2) do
-          patch :update, id: @cohort.id, cohort: {teachers: teacher_params}
+          assert_no_difference('@cohort.districts.count') do
+            patch :update, id: @cohort.id, cohort: {teachers: teacher_params}
+            end
```

Comment:
Lint/BlockAlignment: `end` at 43, 12 is not aligned with `assert_no_difference('@cohort.districts.count') do` at 41, 10